lib/rucio/common/constants.py

```diff
@@ -33,3 +35,10 @@
               's3': ['https', 'davs', 's3']}
 
 SUPPORTED_PROTOCOLS = ['gsiftp', 'srm', 'root', 'davs', 'http', 'https', 'file', 's3', 's3+rucio', 's3+https', 'storm']
+
+FTS_STATE_NAMED_TUPLE = namedtuple('FTS_STATE', ['SUBMITTED', 'READY', 'ACTIVE', 'FAILED', 'FINISHED', 'FINISHEDDIRTY',
+                                                 'CANCELED'])
+FTS_STATE = FTS_STATE_NAMED_TUPLE('SUBMITTED', 'READY', 'ACTIVE', 'FAILED', 'FINISHED', 'FINISHEDDIRTY', 'CANCELED')
```

Comment:
what am I missing here that you do the two steps with the named tuple? can't you just contract it?

Reply:
It can be done in one line. I thought this way is more clearer. Is it okay to do it in one line?

Reply:
I think it's fine to contract. It's not overly complicated and constrained to this file. Just play with the indents to make it obvious.

Reply:
Hi @mlassnig. The request changes have been made.